src/flamegraph/mod.rs

```diff
@@ -248,7 +248,12 @@ impl<'a> Options<'a> {
         } else {
             0
         };
-        self.font_size * 3 + subtitle_height
+        if self.direction == Direction::Straight {
+            self.font_size * 3 + subtitle_height
+        } else {
+            // Inverted (icicle) mode, put the details on top:
+            self.font_size * 4 + subtitle_height + 4
+        }
```

Comment:
I think `ypad2` also has to change to _not_ include padding for the details at the bottom.